src/main/java/uk/gov/hmcts/reform/roleassignmentrefresh/domain/service/process/RefreshJobsOrchestrator.java

```diff
@@ -71,5 +82,12 @@ private void sendJobToORMService(Long jobId, UserRequest userRequest) {
             throw new UnprocessableEntityException(responseEntity.toString());
         }
     }
+
+    public void sendGetUserCountToRASService() {
```

Comment:
Can we rename.  e.g. `triggerRASUserCount'

Reply:
Renamed as suggested